smrf/data/hrrr_grib.py

```diff
@@ -1,14 +1,13 @@
-import logging
-
 import numpy as np
 import pandas as pd
 from weather_forecast_retrieval.hrrr import HRRR
 
 from smrf.envphys.solar.cloud import get_hrrr_cloud
 from smrf.envphys.vapor_pressure import rh2vp
+from .gridded_input import GriddedInput
```

Comment:
Just my style, but I like having the full path to be verbose.

Reply:
Thinking behind a relative path here is to hint that the base class is on the same level. I can go either way.

Reply:
The `image_data` class is imported with the full [path](https://github.com/USDA-ARS-NWRC/smrf/blob/037d17a1b8920abb16cec90e3bc9ec339fbd1e23/smrf/distribute/air_temp.py#L1). Perhaps full path to keep it stylistically the same within SMRF but I understand the reasoning.